services/apis/etl/etlQuestions.js

```diff
@@ -0,0 +1,17 @@
+const chai = require('chai');
+const expect = chai.expect;
+
+/**
+ * fence Questions
+ */
+module.exports = {
+  /**
+   * Asserts a res has url property
+   * @param index
+   * @returns {boolean}
+   */
+  hasVersionIncreased(index, previousVersion) {
+    let no = index.match(RegExp('.*?([0-9]+)$'))
+    expect(Number.parseInt(no[1])).to.equal(previousVersion + 1);
```

Comment:
for all `expect` calls, can you add a description as the second argument so we know why it fails? This applies to all other `expect` calls in this PR too

for example: `expect(Number.parseInt(no[1]), 'version has not increased').` Otherwise, we'll just get errors in the tests like `1 is not equal to 2`

the documentation for this feature is kinda scattered throughout the `chai` docs https://www.chaijs.com/api/bdd/

Reply:
this has been a source of frustration with the current tests and output, we should try to be more diligent about making sure the tests are super descriptive so we know exactly what failed just by looking at jenkins